ContentPatcher/Framework/ContentPatcherAPI.cs

```diff
@@ -97,6 +103,21 @@ public IManagedConditions ParseConditions(IManifest manifest, IDictionary<string
             return this.ParseConditionsImpl(manifest, conditions, formatVersion, assumeModIds);
         }
 
+        public IManagedValue ParseValues(IManifest manifest, string rawValue, ISemanticVersion formatVersion, string[]? assumeModIds = null)
+        {
+            // validate lifecycle
+            if (!this.IsConditionsApiReady)
+                throw new InvalidOperationException($"'{manifest.Name}' accessed Content Patcher's conditions API before it was ready to use. (For mod authors: see the documentation on {nameof(IContentPatcherAPI)}.{nameof(IContentPatcherAPI.IsConditionsApiReady)} for details.)");
+
+            // validate dependency on Content Patcher
+            if (!manifest.HasDependency(this.ContentPatcherID, out ISemanticVersion? minVersion, canBeOptional: false))
+                throw new InvalidOperationException($"'{manifest.Name}' must list Content Patcher as a required dependency in its manifest.json to access the values API.");
+            if (minVersion == null || minVersion.IsOlderThan("1.22.0")) // TODO Change me
```

Comment:
This would be `2.1.0`, assuming it's added in the next Content Patcher major update.

Reply:
I left that for you to change since I wasn't sure when you'd want to release it, but I can go ahead and change it to that.